### PR TITLE
cell2mol as dependency, cif2cell as input, update visualization and instructions added

### PR DESCRIPTION
**Pull request by OsHCuellar**
The major changes are:

- Add cell2mol as a dependency (pinned version with spin prediction). 
- Add cif2cell output as input. 
- Update cel and mol visualization. 
- Add instructions.

The comments of the previous pull request where included too.

### CODE REVIEW

File: requirements.txt
Comment (liam-o-marsh) [inline requirements.txt:1]: do we really need to include (some of) cell2mol's dependencies here?

File: user_templates/instructions.html
Comment (liam-o-marsh) [inline user_templates/instructions.html:1]: it's weird to have three places where the instructions are written, even though there are some light changes in the styling between. It doesn't need to be fixed right now, but it feel like doing so will make headaches smaller later on.

**Review by liam-o-marsh — APPROVED** [review]

**Comment by liam-o-marsh** [comment]
…what is this giant commit?
I think this could have been at least 5 different commits.

File: requirements.txt
Comment (OsHCuellar) [reply requirements.txt:1]: I am not sure why, but when I removed the dependencies from there, it crashes

File: user_templates/instructions.html
Comment (liam-o-marsh) [reply user_templates/instructions.html:1]: no, I meant the fact that they are written multiple times in the code itself. There must be a way to use templates to only have the version in the instructions.html template, and have them rendered in all three steps of the process

Comment (liam-o-marsh) [reply user_templates/instructions.html:1]: thanks!

**Review by liam-o-marsh — APPROVED** [review]

**Pull request merged**